en/docs/get-started/tutorials/create-your-first-rest-api-proxy.md

```diff
@@ -1,94 +1,121 @@
+    |  **Field**    | **Value**                                   |
+    |---------------|---------------------------------------------|
+    | **Name**      | `HR API`                                    |
+    | **Base Path** | `<DEFAULT_VALUE>/ abc-hr`                   |
+    | **Version**   | `1.0.0`                                     |
+    | **Target**    | `https://samples.choreoapps.dev/company/hr` |
 
-6. Select the **Input Type** and provide the OpenAPI URL as follows:
- 
-    | **Field** | **Value** |
-    |-----------------|-----------------------|
-    | **Input Type** | OpenAPI URL |
-    | **OpenAPI URL** | `https://petstore3.swagger.io/api/v3/openapi.json` |
+7. Click **Create**. 
+
+The API proxy opens on a separate page.
+
+## Step 2: Developing the REST API Proxy
+
+Let's develop the REST API proxy by adding an operation. To do this, follow the steps given below:
+
+1. In the left navigation, click **Develop**.
+
+2. In the **Resources** tab, add a new resource to fetch the department ID as follows:
+
+    1. Select **GET** as the HTTP verb.
 
-7. Click **Next**.
+    2. Enter `/department/{departmentId}` as the URI pattern.
 
-8. Review the information (API Name, API Version, API Base Path, Endpoint) that Choreo extracted from the OpenAPI. You can make any changes to the fields if you require. Click **Create** to create your REST API Proxy.
+    3. To add the new resource, click **+**.
 
-## Step 2: Deploy the REST API Proxy
+    4. Once the resource is added, expand it, and then enter a name and a description for it as given below:
 
-Deploying the REST API proxy makes it invokable. Choreo maintains two environments by default: development and production. [Learn more](../../develop/components/api-proxies/api-proxy#choreo-environments). 
+        | **Field**        | **Description**                      |
+        |------------------|--------------------------------------|
+        | **Operation ID** | `findDepartment`                     |
+        | **Description**  | `Find a department by Department ID` |
 
-1. Return to the Choreo Console. From the left navigation, click **Deploy**. 
+    5. Click **Save**.
 
-2. Let's deploy the REST API proxy to the development environment. Click **Config & Deploy** in the **Build Area** column.
+3. Click **API Definition**. You will see the new operation you added in the OpenAPI definition.
 
-3. Once you have deployed it to the development environment, you can test it. When you are ready to take it to production, you can come back to this view and promote it by clicking on the **Promote** button on the **Development** column.
+## Step 3: Deploy the REST API Proxy
 
-    ![Promote REST API proxy](../../assets/img/tutorials/promote-rest-api-proxy.png){.cInlineImage-threeQuarter}
+Deploying the REST API proxy makes it invokable. By default, Choreo maintains two environments for development and production. [Learn more](../../develop/components/api-proxies/api-proxy#choreo-environments). 
```

Comment:
Shall we add a sentence to say what we will be doing? For e.g. for the purpose of this tutorial let's deploy on the redevelopment environemnt.

Reply:
Done